packages/components/src/color-palette/test/index.tsx

```diff
@@ -225,6 +225,9 @@ describe( 'ColorPalette', () => {
 
 		render( <ControlledColorPalette /> );
 
+		const colorName = EXAMPLE_COLORS[ 0 ].name;
+		const colorCode = EXAMPLE_COLORS[ 0 ].color;
```

Comment:
Could be simplified by using destructuring with aliases:

```suggestion
		const { name: colorName, color: colorCode } = EXAMPLE_COLORS[ 0 ];
```

Reply:
Great suggestion!